multimodal-search-tirg/app.py

```diff
@@ -57,18 +58,27 @@ def print_result(resp):
         images.append(image)
     plot_topk_images(images)
 
+def index_generator(data_path, num_docs):
+    for buffer in input_files(data_path, True, num_docs, None, 'rb'):
+        with Document() as doc:
+            doc.buffer = buffer
+            doc.mime_type = 'image/jpeg'
+        yield doc
+
 def query_generator(image_paths, text_queries):
     for image_path, text in zip(image_paths, text_queries):
         with open(image_path, 'rb') as fp:
             buffer = fp.read()
-        yield MultimodalDocument(modality_content_map={'image': buffer, 'text': text})
+        with MultimodalDocument(modality_content_map={'image': buffer, 'text': text}) as md:
+            md.mime_type = 'image/jpeg'
```

Comment:
why multimodal has a mime type? it should have none

Reply:
removed